src/pg/sql/41_observatory_augmentation.sql

```diff
@@ -760,16 +760,13 @@ BEGIN
               || ') val_' || colid, ', ')
             || '
             FROM _procgeoms_' || Coalesce(geom_tablename || '_' || geom_geomref_colname, api_method) || ' _procgeoms ' ||
-              Coalesce(', ' || String_Agg(DISTINCT
-                  Coalesce('observatory.' || numer_tablename,
+              Coalesce(String_Agg(DISTINCT
```

Comment:
Yes, you're right. I removed it because left join doesn't work with comma but lateral is failing. 

I don't know what is the api_method part, which is the one generating the LATERAL join but I assume we have to maintain it.

Fixing this problem...